diff --git a/node_modules/react-native-google-places/android/build.gradle b/node_modules/react-native-google-places/android/build.gradle
index bbb42c6..4d0f745 100644
--- a/node_modules/react-native-google-places/android/build.gradle
+++ b/node_modules/react-native-google-places/android/build.gradle
@@ -49,4 +49,8 @@ dependencies {
     implementation "com.android.support:appcompat-v7:${supportLibVersion}"
     implementation "com.android.support:support-v4:${supportLibVersion}"
     implementation "com.google.code.findbugs:jsr305:3.0.2"
+    implementation('com.google.android.libraries.places:places:1.1.0') {
+        exclude group: 'com.android.volley'
+    }
+    implementation 'com.android.volley:volley:1.2.0'
 }
diff --git a/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.h b/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.h
index 8991707..0b79ec9 100644
--- a/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.h
+++ b/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.h
@@ -1,6 +1,5 @@
 #import <Foundation/Foundation.h>
 #import <GooglePlaces/GooglePlaces.h>
-
 @interface NSMutableDictionary (GMSPlace)
 
 + (instancetype)dictionaryWithGMSPlace:(GMSPlace*)place;
diff --git a/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.m b/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.m
index bd37cf1..1c02255 100644
--- a/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.m
+++ b/node_modules/react-native-google-places/ios/NSMutableDictionary+GMSPlace.m
@@ -48,12 +48,12 @@ + (instancetype)dictionaryWithGMSPlace:(GMSPlace*)place
         placeData[@"rating"] = [NSNumber numberWithDouble:place.rating];
     }
     
-    if (place.viewport) {
+    if (place.viewportInfo) {
         NSMutableDictionary *viewportMap = [[NSMutableDictionary alloc] init];
-        viewportMap[@"latitudeNE"] = [NSNumber numberWithDouble:place.viewport.northEast.latitude];
-        viewportMap[@"longitudeNE"] = [NSNumber numberWithDouble:place.viewport.northEast.longitude];
-        viewportMap[@"latitudeSW"] = [NSNumber numberWithDouble:place.viewport.southWest.latitude];
-        viewportMap[@"longitudeSW"] = [NSNumber numberWithDouble:place.viewport.southWest.longitude];
+        viewportMap[@"latitudeNE"] = [NSNumber numberWithDouble:place.viewportInfo.northEast.latitude];
+        viewportMap[@"longitudeNE"] = [NSNumber numberWithDouble:place.viewportInfo.northEast.longitude];
+        viewportMap[@"latitudeSW"] = [NSNumber numberWithDouble:place.viewportInfo.southWest.latitude];
+        viewportMap[@"longitudeSW"] = [NSNumber numberWithDouble:place.viewportInfo.southWest.longitude];
 
         placeData[@"viewport"] = viewportMap;
     }
diff --git a/node_modules/react-native-google-places/ios/RNGooglePlaces.m b/node_modules/react-native-google-places/ios/RNGooglePlaces.m
index 4b8f3f9..d462b87 100644
--- a/node_modules/react-native-google-places/ios/RNGooglePlaces.m
+++ b/node_modules/react-native-google-places/ios/RNGooglePlaces.m
@@ -13,7 +13,6 @@
 @interface RNGooglePlaces() <CLLocationManagerDelegate>
 
 @property (strong, nonatomic) CLLocationManager *locationManager;
-@property GMSAutocompleteBoundsMode boundsMode;
 
 @end
 
@@ -40,8 +39,6 @@ - (instancetype)init
         _instance = self;
         self.locationManager = [[CLLocationManager alloc] init];
         self.locationManager.delegate = self;
-        
-        self.boundsMode = kGMSAutocompleteBoundsModeBias;
     }
     
     return self;
@@ -71,9 +68,14 @@ - (void)dealloc
         NSDictionary *locationRestriction = [RCTConvert NSDictionary:options[@"locationRestriction"]];
         
         
-        GMSCoordinateBounds *autocompleteBounds = [self getBounds:locationBias andRestrictOptions:locationRestriction];
+        [self applyLocationOptionsWithBiasOptions:locationBias
+                                  restrictOptions:locationRestriction
+                                         toFilter:autocompleteFilter];
 
-        [acController openAutocompleteModal: autocompleteFilter placeFields: selectedFields bounds: autocompleteBounds boundsMode: self.boundsMode resolver: resolve rejecter: reject];
+        [acController openAutocompleteModal: autocompleteFilter
+                                placeFields: selectedFields
+                                   resolver: resolve
+                                   rejecter: reject];
     }
     @catch (NSException * e) {
         reject(@"E_OPEN_FAILED", @"Could not open modal", [self errorFromException:e]);
@@ -93,13 +95,13 @@ - (void)dealloc
     NSDictionary *locationBias = [RCTConvert NSDictionary:options[@"locationBias"]];
     NSDictionary *locationRestriction = [RCTConvert NSDictionary:options[@"locationRestriction"]];
     
-    GMSCoordinateBounds *autocompleteBounds = [self getBounds:locationBias andRestrictOptions:locationRestriction];
+    [self applyLocationOptionsWithBiasOptions:locationBias
+                              restrictOptions:locationRestriction
+                                     toFilter:autocompleteFilter];
     
     GMSAutocompleteSessionToken *token = [[GMSAutocompleteSessionToken alloc] init];
     
     [[GMSPlacesClient sharedClient] findAutocompletePredictionsFromQuery:query
-                                               bounds:autocompleteBounds
-                                               boundsMode:self.boundsMode
                                                filter:autocompleteFilter
                                                sessionToken:token
                                              callback:^(NSArray<GMSAutocompletePrediction *> * _Nullable results, NSError *error) {
@@ -273,7 +275,9 @@ - (GMSPlaceField) getSelectedFields:(NSArray *)fields isCurrentOrFetchPlace:(Boo
     return GMSPlaceFieldAll;
 }
 
-- (GMSCoordinateBounds *) getBounds: (NSDictionary *)biasOptions andRestrictOptions: (NSDictionary *)restrictOptions
+- (void)applyLocationOptionsWithBiasOptions:(NSDictionary *)biasOptions
+                            restrictOptions:(NSDictionary *)restrictOptions
+                                   toFilter:(GMSAutocompleteFilter *)autocompleteFilter
 {
     double biasLatitudeSW = [[RCTConvert NSNumber:biasOptions[@"latitudeSW"]] doubleValue];
     double biasLongitudeSW = [[RCTConvert NSNumber:biasOptions[@"longitudeSW"]] doubleValue];
@@ -288,26 +292,16 @@ - (GMSCoordinateBounds *) getBounds: (NSDictionary *)biasOptions andRestrictOpti
     if (biasLatitudeSW != 0 && biasLongitudeSW != 0 && biasLatitudeNE != 0 && biasLongitudeNE != 0) {
         CLLocationCoordinate2D neBoundsCorner = CLLocationCoordinate2DMake(biasLatitudeNE, biasLongitudeNE);
         CLLocationCoordinate2D swBoundsCorner = CLLocationCoordinate2DMake(biasLatitudeSW, biasLongitudeSW);
-        GMSCoordinateBounds *bounds = [[GMSCoordinateBounds alloc] initWithCoordinate:neBoundsCorner
-                                                                        coordinate:swBoundsCorner];
-
-        return bounds;
+        autocompleteFilter.locationBias = GMSPlaceRectangularLocationOption(neBoundsCorner, swBoundsCorner);
+        return;
     }  
 
     if (restrictLatitudeSW != 0 && restrictLongitudeSW != 0 && restrictLatitudeNE != 0 && restrictLongitudeNE != 0) {
         CLLocationCoordinate2D neBoundsCorner = CLLocationCoordinate2DMake(restrictLatitudeNE, restrictLongitudeNE);
         CLLocationCoordinate2D swBoundsCorner = CLLocationCoordinate2DMake(restrictLatitudeSW, restrictLongitudeSW);
-        GMSCoordinateBounds *bounds = [[GMSCoordinateBounds alloc] initWithCoordinate:neBoundsCorner
-                                                                        coordinate:swBoundsCorner];
-        
-        self.boundsMode = kGMSAutocompleteBoundsModeRestrict;
-        
-        return bounds;
+        autocompleteFilter.locationRestriction = GMSPlaceRectangularLocationOption(neBoundsCorner, swBoundsCorner);
     }
-    
-    return nil;
 }
 
 
 @end
-
diff --git a/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.h b/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.h
index ca969fd..5ebeb12 100644
--- a/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.h
+++ b/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.h
@@ -11,8 +11,6 @@
 
 - (void)openAutocompleteModal: (GMSAutocompleteFilter *)autocompleteFilter
                   placeFields: (GMSPlaceField)selectedFields
-                       bounds: (GMSCoordinateBounds *)autocompleteBounds
-                   boundsMode: (GMSAutocompleteBoundsMode)autocompleteBoundsMode
                      resolver: (RCTPromiseResolveBlock)resolve
                      rejecter: (RCTPromiseRejectBlock)reject;
                      
diff --git a/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.m b/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.m
index ea60607..f1236d8 100644
--- a/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.m
+++ b/node_modules/react-native-google-places/ios/RNGooglePlacesViewController.m
@@ -26,18 +26,14 @@ - (instancetype)init
 
 - (void)openAutocompleteModal: (GMSAutocompleteFilter *)autocompleteFilter
                     placeFields: (GMSPlaceField)selectedFields
-                       bounds: (GMSCoordinateBounds *)autocompleteBounds
-                       boundsMode: (GMSAutocompleteBoundsMode)autocompleteBoundsMode
                      resolver: (RCTPromiseResolveBlock)resolve
-                     rejecter: (RCTPromiseRejectBlock)reject;
+                     rejecter: (RCTPromiseRejectBlock)reject
 {
     _resolve = resolve;
     _reject = reject;
     
     GMSAutocompleteViewController *viewController = [[GMSAutocompleteViewController alloc] init];
     viewController.autocompleteFilter = autocompleteFilter;
-    viewController.autocompleteBounds = autocompleteBounds;
-    viewController.autocompleteBoundsMode = autocompleteBoundsMode;
     viewController.placeFields = selectedFields;
 	viewController.delegate = self;
     UIViewController *topController = [self getTopController];
diff --git a/node_modules/react-native-google-places/react-native-google-places.podspec b/node_modules/react-native-google-places/react-native-google-places.podspec
index 7445c14..a82c0df 100644
--- a/node_modules/react-native-google-places/react-native-google-places.podspec
+++ b/node_modules/react-native-google-places/react-native-google-places.podspec
@@ -19,6 +19,6 @@ Pod::Spec.new do |s|
   s.compiler_flags = '-DHAVE_GOOGLE_MAPS=1', '-fno-modules'
 
   s.dependency 'React'
-  s.dependency 'GooglePlaces', '~> 3.1.0'
-  s.dependency 'GoogleMaps', '~> 3.1.0'
+  s.dependency 'GooglePlaces', '9.4.0'
+  s.dependency 'GoogleMaps', '9.4.0'
 end
